vizalerts.py

```diff
@@ -1045,29 +1044,41 @@ def get_password_from_file(password):
         sys.exit(1)
 
 
-def find_vizimages_in_body(data, has_email_header, has_email_footer):
-    """Returns list of all found occurences of VIZ_IMAGE() in email body text"""
+
+def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
+    # updated 20160223 Jonathan Drummey
+    # function call now includes viewurlsuffix, return is now a list of viewnam
```

Comment:
I'm not sure I understand why viewurlsuffix was added as a parameter? It doesn't seem to be referenced in the function.